### PR TITLE
BusinessDashboard Theme | SectionHelper - override styles

### PR DESCRIPTION
**Pull request by adamg-wix**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [x] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [x] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [x] Visual test

Screen Shot 2021-03-25 at 11 02 55

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6950/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6950

File: packages/wix-style-react/src/SectionHelper/test/SectionHelper.visual.js [hunk 1]
```diff
@@ -12,6 +12,45 @@ const commonProps = {
...
+            </Cell>
+            <Cell>
+              <SectionHelper {...commonProps} showCloseButton={false} />
+            </Cell>
+          </Layout>
+        </div>,
+      ),
+    );
+};
+
```
Comment (adicicurel) [inline packages/wix-style-react/src/SectionHelper/test/SectionHelper.visual.js:53]: runTests should wrap the original testing

File: packages/wix-style-react/src/Themes/businessDashboard/components/SectionHelper.st.css [hunk 2]
```diff
@@ -0,0 +1,74 @@
+:import {
+  -st-from: '../../../Foundation/stylable/typography.st.css';
+  -st-named: text-tiny-thin, text-tiny-bold
+}
+
+:import {
+  -st-from: "../foundations/spacing.st.css";
+  -st-named: SP1, SP2, SP3, SP4, SP5, SP6, SP7, SP8;
```
Comment (adicicurel) [inline packages/wix-style-react/src/Themes/businessDashboard/components/SectionHelper.st.css:8]: Import only what in use

File: packages/wix-style-react/src/Themes/businessDashboard/components/SectionHelper.st.css [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  border-radius: value(SP1);
+}
+
+.root:withCloseBtn {
+  padding: value(SP3) value(SP8) value(SP3) value(SP5);
+}
+
+:global([dir='rtl']) .root:withCloseBtn {
+  padding: value(SP3) value(SP5) value(SP3) value(SP8);
+}
```
Comment (adicicurel) [inline packages/wix-style-react/src/Themes/businessDashboard/components/SectionHelper.st.css:41]: is it part of the theme according to UX?

File: packages/wix-style-react/src/Themes/businessDashboard/components/SectionHelper.st.css [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+.root:appearance(warning) {
+  background-color: value(Y50);
+  border-color: transparent;
```
Comment (adicicurel) [inline packages/wix-style-react/src/Themes/businessDashboard/components/SectionHelper.st.css:45]: border: none ?

File: packages/wix-style-react/src/Themes/businessDashboard/components/SectionHelper.st.css [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+.root:appearance(standard) {
+  background-color: value(B50);
+  border-color: transparent;
```
Comment (adicicurel) [inline packages/wix-style-react/src/Themes/businessDashboard/components/SectionHelper.st.css:50]: same

File: packages/wix-style-react/src/SectionHelper/test/SectionHelper.visual.js (hunk 1 above)
Comment (adamg-wix) [reply packages/wix-style-react/src/SectionHelper/test/SectionHelper.visual.js:53]: If i do that wouldn't it remove the non-themed tests ?

File: packages/wix-style-react/src/Themes/businessDashboard/components/SectionHelper.st.css [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+.root::title Text {
+  -st-mixin: text-tiny-bold;
+}
+
+.root::content Text { 
+  -st-mixin: text-tiny-thin;
+}
+
+.root::close {
+  position: absolute;
```
Comment (adicicurel) [inline packages/wix-style-react/src/Themes/businessDashboard/components/SectionHelper.st.css:71]: why changing its position?

File: packages/wix-style-react/src/Themes/businessDashboard/foundations/colors.st.css [hunk 7]
```diff
@@ -43,11 +43,16 @@
...
+  R40: #FDE3E1;
   R50: #FDECEB;
 
   /* Orange */
   O10: #F96132;
 
+  /* Yellow */
+  Y30: #FFD988;
+  Y50: #FFF7E2;
+
```
Comment (adicicurel) [inline packages/wix-style-react/src/Themes/businessDashboard/foundations/colors.st.css:55]: were those approved with Ayelet Livne?

File: packages/wix-style-react/src/Themes/businessDashboard/foundations/colors.st.css [hunk 8]
```diff
@@ -43,11 +43,16 @@
   /* Destructive */
   R10: #EE4437;
   R20: #F1695F;
+  R40: #FDE3E1;
```
Comment (adicicurel) [inline packages/wix-style-react/src/Themes/businessDashboard/foundations/colors.st.css:46]: were this approved with Ayelet Livne?

File: packages/wix-style-react/src/Themes/businessDashboard/components/SectionHelper.st.css (hunk 3 above)
Comment (adamg-wix) [reply packages/wix-style-react/src/Themes/businessDashboard/components/SectionHelper.st.css:41]: I just assumed i have to support RTL as well. it's just the reverse of the regular closeButton padidng.

**Review by adicicurel — CHANGES_REQUESTED** [review]
comments inside.

File: packages/wix-style-react/src/Themes/businessDashboard/foundations/colors.st.css (hunk 8 above)
Comment (adamg-wix) [reply packages/wix-style-react/src/Themes/businessDashboard/foundations/colors.st.css:46]: see https://jira.wixpress.com/browse/OSA-306

File: packages/wix-style-react/src/Themes/businessDashboard/foundations/colors.st.css (hunk 7 above)
Comment (adamg-wix) [reply packages/wix-style-react/src/Themes/businessDashboard/foundations/colors.st.css:55]: https://jira.wixpress.com/browse/OSA-306

File: packages/wix-style-react/src/Themes/businessDashboard/components/SectionHelper.st.css (hunk 6 above)
Comment (adamg-wix) [reply packages/wix-style-react/src/Themes/businessDashboard/components/SectionHelper.st.css:71]: Added

File: packages/wix-style-react/src/Themes/businessDashboard/components/SectionHelper.st.css (hunk 2 above)
Comment (adamg-wix) [reply packages/wix-style-react/src/Themes/businessDashboard/components/SectionHelper.st.css]: done

File: packages/wix-style-react/src/Themes/businessDashboard/components/SectionHelper.st.css (hunk 4 above)
Comment (adamg-wix) [reply packages/wix-style-react/src/Themes/businessDashboard/components/SectionHelper.st.css]: done

File: packages/wix-style-react/src/Themes/businessDashboard/components/SectionHelper.st.css (hunk 5 above)
Comment (adamg-wix) [reply packages/wix-style-react/src/Themes/businessDashboard/components/SectionHelper.st.css]: done

**Review by adicicurel — APPROVED** [review]

File: packages/wix-style-react/src/Themes/businessDashboard/components/SectionHelper.st.css (hunk 3 above)
Comment (mykas) [reply packages/wix-style-react/src/Themes/businessDashboard/components/SectionHelper.st.css:41]: Please don't use `:global` params. In order to support rtl based padding you can use `padding-inline` https://developer.mozilla.org/en-US/docs/Web/CSS/padding-inline

**Review by mykas — COMMENTED** [review]
See my reply in comments of `:global` params usage.

File: packages/wix-style-react/src/Themes/businessDashboard/components/SectionHelper.st.css (hunk 3 above)
Comment (adamg-wix) [reply packages/wix-style-react/src/Themes/businessDashboard/components/SectionHelper.st.css:41]: @mykas this is how it is done in the original component as well i was just following it's lead :)
I worry that if i use a different implementation here the output might not be the same.

Comment (mykas) [reply packages/wix-style-react/src/Themes/businessDashboard/components/SectionHelper.st.css:41]: oh thats true. Allright

**Pull request merged**